doc_source/guide_waiters.rst

```diff
@@ -19,7 +19,12 @@ Waiters in the |sdk-php| Version 3
 Waiters help make it easier to work with *eventually consistent* systems by
 providing an abstracted way to wait until a resource enters into a particular
 state by polling the resource. You can find a list of the waiters supported by
-a client by viewing the API documentation of a service client.
+a client by viewing the API documentation for a single version of a service client.
+To navigate there, go to the client's page in our documentation and click on the
+hyperlink that is in the date of the version and scroll down to the 'Waiters' section.

```

Comment:
"click on the hyperlink that is in the date of the version" may be a bit confusing. Perhaps something like "navigate to a specific API version (represented by a date)"?

Reply:
Updated the language to "To navigate there, go to the client's
page in the API documentation and navigate to the specific version number
(represented by a date) and scroll down to the 'Waiters' section."